composer.json

```diff
@@ -14,7 +14,8 @@
     "require": {
         "php": "^5.6|^7.0",
         "symfony-cmf/routing": "^2.0",
-        "symfony/framework-bundle": "^2.8|^3.0"
+        "symfony/framework-bundle": "^2.8|^3.0",
+        "symfony/twig-bundle": "^2.8|^3.0"
```

Comment:
are you sure we need the twig bundle? would twig/twig not be enough?

Reply:
ah sorry, we need the bundle because we want the twig.loader service. this change is fine imho.